enigma/src/main/java/cuchaz/enigma/source/cfr/CfrDecompiler.java

```diff
@@ -19,6 +20,7 @@
 import org.benf.cfr.reader.util.collections.ListFactory;
 import org.benf.cfr.reader.util.getopt.Options;
 import org.benf.cfr.reader.util.getopt.OptionsImpl;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
Not for this PR but noticed it here; should prove move it to jetbrains annotations at somepoint.